contract/src/utils.rs

```diff
@@ -738,3 +742,17 @@ pub(crate) fn get_reporting_mode() -> OwnerReverseLookupMode {
     .try_into()
     .unwrap_or_revert()
 }
+
+// Initializes events-releated named keys and records all event schemas.
+pub fn init_events() {
+    let schemas = Schemas::new()
```

Comment:
Hi, thanks for your PR.

IS there a way to have a `Schemas::default()` here instead of having to specify following with ?

Thanks

Reply:
`Default` is implemented, so `Schema::default()` is possible. Being honest I like `new`. This way it feels more like `Vec::new()` or `BTreeMap::new()`, which is much more used than `Vec::default()`.

Reply:
Hi thanks yes I was trying to see if we could simplify this new + struct as generic to a simple ::default() for the consumer and have the external crate serve a default set of events already. But I see in your integrations tests (casper-event-standard/integration-tests/bin/event_initializer.rs) that actually structs Mint Transfer etc are not coming from the lib but only in tests, so the implementer has to specify the full list ok.